db/psychologists.js

```diff
@@ -136,7 +154,8 @@ module.exports.savePsychologistInPG = async function savePsychologistInPG(psyLis
     }
   });
 
-  const query = await Promise.all(upsertArray);
+  upsertArray.push(knex(psychologistsTable).insert(psychologistsToInsert));
```

Comment:
What if the insert query fails? Before we had a try catch logging the error and resolving the promise. Here, the promise will end with an error, will we know by any mean that the insertions failed?

Reply:
it fails properly 
![image](https://user-images.githubusercontent.com/5389328/122423454-38316900-cf8e-11eb-96d4-4299ab08cb0d.png)